source/editors/ChartingState.hx

```diff
@@ -73,8 +73,9 @@ class ChartingState extends MusicBeatState
 		['', "Nothing. Yep, that's right."],
 		['Hey!', "Plays the \"Hey!\" animation from Bopeebo,\nValue 1: BF = Only Boyfriend, GF = Only Girlfriend,\nSomething else = Both.\nValue 2: Custom animation duration,\nleave it blank for 0.6s"],
 		['Set GF Speed', "Sets GF head bopping speed,\nValue 1: 1 = Normal speed,\n2 = 1/2 speed, 4 = 1/4 speed etc.\nUsed on Fresh during the beatbox parts.\n\nWarning: Value must be integer!"],
-		['Blammed Lights', "Value 1: 0 = Turn off, 1 = Blue, 2 = Green,\n3 = Pink, 4 = Red, 5 = Orange, Anything else = Random.\n\nNote to modders: This effect is starting to get \nREEEEALLY overused, this isn't very creative bro smh."],
-		['Kill Henchmen', "For Mom's songs, don't use this please, i love them :("],
+		['Change Hurt Amount', "Changes the amount of damage Hurt Notes deal\nValue 1: Full notes (0.3 is default)\nValue 2: Sustain tail (0.1 is default)"],
+		['Blammed Lights', "Value 1: 0 = Turn off, 1 = Red, 2 = Black Anything else = Random"],
```

Comment:
BLAMMED LIGHTS !!

Reply:
-- WHY WOULD THEY RANDOMLY REMOVE COLORS??????? WHY WOULD YOU MAKE IT NOT WORK WITH THE ONE STAGE WHERE IT'S ACTUALLY ACCEPTABLE TO USE BLAMMED LIGHTS??????